website/content/en/preview/upgrading/upgrade-guide.md

```diff
@@ -37,6 +37,75 @@ kubectl apply -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef
 WHEN CREATING A NEW SECTION OF THE UPGRADE GUIDANCE FOR NEWER VERSIONS, ENSURE THAT YOU COPY THE BETA API ALERT SECTION FROM THE LAST RELEASE TO PROPERLY WARN USERS OF THE RISK OF UPGRADING WITHOUT GOING TO 0.32.x FIRST
 -->
 
+### Upgrading to `1.0.0`+
+
+{{% alert title="Warning" color="warning" %}}
+`1.0.0`+ supports Karpenter v1 and v1beta1 APIs and will not work with earlier Provisioner, AWSNodeTemplate or Machine alpha APIs. Do not upgrade to `1.0.0`+ without first [upgrading to `0.32.x`]({{<ref "#upgrading-to-0320" >}}) or later. This version supports both the v1beta1 and v1 APIs, allowing you to migrate all of your existing APIs to v1beta1 APIs without experiencing downtime.
+{{% /alert %}}
+
+Before you begin upgrading to `1.0.0`, you should know that:
+
+* Every Karpenter upgrade from pre-v1.0.0 versions must go through an upgrade to minor version `v1.0.0`.
+* You must be on a version of Karpenter that supports NodePools, NodeClaims, and NodeClasses (`0.33.0`+ Karpenter versions support v1beta1 APIs).
+* You must complete all `Before upgrading to 1.0.0` Prerequisites.
+* You must complete all `Before upgrading to `1.1.0` Prerequisites before upgrading to `1.1.0` later. Although v1beta1 yaml files should work for `v1.0.0`, the will fail with the release of `v1.1.0`.
+
+#### Prerequisites
+
+Consider two different categories of `1.0.0` prerequisites: those you must do before `1.0.0` upgrades and those you must do before `1.1.0` upgrades.
+
+
+Before upgrading to `1.0.0`:
+
+* **Deprecated annotations, labels and tags are removed for v1.0.0**: For v1, `karpenter.sh/do-not-consolidate` (annotation), `karpenter.sh/do-not-evict
+(annotation)`, and `karpenter.sh/managed-by` (tag) all have support removed.
+The `karpenter.sh/managed-by`, which currently stores the cluster name in its value, is replaced by `eks:eks-cluster-name`, to allow
+for [EKS Pod Identity ABAC policies](https://docs.aws.amazon.com/eks/latest/userguide/pod-id-abac.html).
+
+* **Zap logging removed**: Support for setting the Zap logging config was deprecated in beta and is now removed for v1. View the [Logging Configuration Section of the v1beta1 Migration Guide]({{<ref "../../v0.32/upgrading/v1beta1-migration#logging-configuration-is-no-longer-dynamic" >}}) for more details.
+
+* **metadataOptions could break workloads**: If you have workload pods that are not using `hostNetworking`, the updated default `metadataOptions`
+could cause your containers to break when you apply new EC2NodeClasses on v1.
+
+* **Ubuntu alias dropped**: If you are using Ubuntu, be aware that Karpenter v1 no longer supports an `alias` for using Ubuntu.
```

Comment:
More details on this: 
"If you're setting AMIFamily: Ubuntu" and have no AMISelectorTerms defined, Karpenter automatically upgrades AMIs for you. Automatic upgrades are not supported for Ubuntu in v1, and you must pin your AMIs before upgrading"